main.py

```diff
@@ -283,16 +290,70 @@ def disable_switch(self, request: Request) -> JSONResponse:
         dpid = request.path_params["dpid"]
         try:
             switch = self.controller.switches[dpid]
+            for _, interface in switch.interfaces.copy().items():
+                if (interface.link and interface.link.is_enabled()):
+                    self.notify_link_enabled_state(interface.link, "disabled")
```

Comment:
I've just realized an interesting case here. When [switch.disable()](https://github.com/kytos-ng/kytos/blob/master/kytos/core/switch.py#L162-L169) is called, it'll also disable its interfaces (historically, it's been transitively this way for a `Switch` entity since it owns each `Interface`). Since a `link` `is_enabled()` takes into account interface `is_enabled()` status too, then `link.status` will be `EntityStatus.DISABLED` too, but here's the interesting point, it's not updating the `db.links` collection. So disabling links and interfaces are being treated like an standalone association, and still makes sense since link has its own enabling state too, but we gotta keep this in mind in the future not to get confused with this. Good thing that during restart it'll also do the right thing since endpoints/interfaces are disabled the link will still be disabled. I'm not seeing any other potential issue due to this, but let me know if you can see any other potential issue implication because of this.

Reply:
I have seen `path` from `mef_eline` relying in this database, [code](https://github.com/kytos-ng/mef_eline/blob/master/models/path.py#L99). But `link_down` disables overlaps this status confirmation. Although maybe there could be a bug related to path giving false information.

Reply:
@Alopalao `GET /links` contents will come from `self.links` in memory, so the status will be `DISABLED` (despite on `links` collection still be enabled, and after a `kytosd` restart it'll still be loading correctly too from what I can follow). Let me know if you've seen any other case that I haven't spotted. Let's leave this PR to get merged on Monday then. Thanks, Aldo.